src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/DesignerTabButton.vb

```diff
@@ -151,28 +151,42 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
                     If parent IsNot Nothing Then
                         parent.OnItemClick(Me, reactivatePage:=True)
                     End If
-                Case Keys.Tab, Keys.Up, Keys.Down
-
-                    If keyCode <> Keys.Tab OrElse (keyData And Keys.Control) <> Keys.Control Then                                   ' Don't process if Ctrl+Tab, but do Process for Ctrl+Up and Ctrl+Down
+                Case Keys.Up
+                    Dim parent As ProjectDesignerTabControl = ParentTabControl
```

Comment:
Now that there are a bunch more cases, should the declaration of `parent` and the `Nothing` check be moved out of the switch? Or are you trying to defer it on purpose?